tests/jeos/image_info.pm

```diff
@@ -0,0 +1,127 @@
+    }
+    my $size_mb = $size / 1024 / 1024;
+    record_info('Image', "Image: $image\nSize: $size_mb");
+
+    # Record this value in the InfluxDB
+    # We only allow this on certain conditions:
+    #  - The job must be executed from OSD or O3
+    #  - The job must contain INFLUXDB_SERVER and _SECRET_INFLUXDB_USER and _SECRET_INFLUXDB_PWD
+    #  - The job shouldn't be a verification run => CASEDIR variable
+    my $influxdb_server = get_var('INFLUXDB_SERVER');
+    my $influxdb_user = get_var('_SECRET_INFLUXDB_USER');
+    my $influxdb_pwd = get_var('_SECRET_INFLUXDB_PWD');
+    #return unless (get_var('CASEDIR') || $influxdb_server || $influxdb_user || $influxdb_pwd); #this should remain before merging
+    return unless ($influxdb_server || $influxdb_user || $influxdb_pwd);    # remove CASEDIR condition to allow running VR before merging
+
+    my $job_url;
+    my $openqa_host = get_required_var('OPENQA_HOSTNAME');
+    if ($openqa_host =~ /openqa1-opensuse|openqa.opensuse.org/) {    # O3 hostname
+        $job_url = 'https://openqa.opensuse.org/tests/' . get_current_job_id();
+    }
+    elsif ($openqa_host =~ /openqa.suse.de/) {    # OSD hostname
+        $job_url = 'https://openqa.suse.de/tests/' . get_current_job_id();
+    }
+
+    return unless ($job_url);
+
+    # Check if the data for this image has been already published before.
+    # This will avoid publishing the same data if a job is restarted.
+    system("echo '-u $influxdb_user:$influxdb_pwd' > $dir/influxdb_conf");
+
+    my $query = 'curl -K ' . $dir . '/influxdb_conf -i -G "http://' . $influxdb_server . ':8086/query?db=data"';
+    $query .= ' --data-urlencode "q=SELECT \"value\" FROM \"image_size\" WHERE \"image\" = \'' . $image . '\'"';
+    my $query_output = `$query 2>&1`;
+    record_info('DB check', "Check if this value exists in the DB.\nquery = $query\nresult = \n$query_output");
+    # Successful query should return 'HTTP/1.1 200 OK'
+    # Empty records will return  '{"results":[{"statement_id":0}]}'
+    # Existing records will return '{"results":[{"statement_id":0,"series":[{"name":"size","columns":["time","value"],"values"' ...
+    if ($query_output !~ /200 OK/) {
+        record_soft_failure("poo#110221 - There has been a problem with the query.");
+    } elsif ($query_output =~ /values/) {
+        record_info('Skip', "Skip pushing the data to the DB. There is already a DB record for this image.");
+    } else {
+        $query = 'curl -K influxdb_conf -i -XPOST "http://' . $influxdb_server . ':8086/write?db=data"';
+        $query .= ' --data-binary "image_size';
+        $query .= ',distri=' . get_required_var('DISTRI');
+        $query .= ',version=' . get_required_var('VERSION');
+        $query .= ',arch=' . get_required_var('ARCH');
+        $query .= ',flavor=' . get_required_var('FLAVOR');
+        $query .= ',build=' . get_required_var('BUILD');
+        $query .= ',image=' . $image;
+        $query .= ',job_url=' . $job_url;
```

Comment:
Why do we need this?

Reply:
It would be good to have the information of which job posted the value. This is just metadata, so the more info we have, the easier to troubleshoot.